modules/tensor_mechanics/include/materials/ComputeAdmissibleState.h

```diff
@@ -0,0 +1,112 @@
+/****************************************************************/
```

Comment:
I hate to make you rename everything, but the more I think about this, the more I don't like this name.  I know that to be precise, the model does much more than compute the stress, but our convention is that all of the models that compute stress have 'Stress' in the name.  We have an awfully large number of models named `Compute*Stress` that compute more than just the stress. 

Also, this name doesn't convey the fact that this is what you use to drive potentially multiple stress update models.  I know that this isn't perfect, but I propose naming this `ComputeMultipleInelasticStress`.  This conveys that it is a driver for potentially multiple inelastic models, and that it is a stress calculator.